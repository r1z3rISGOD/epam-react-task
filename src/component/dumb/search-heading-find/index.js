import {SearchHeadingNetflix} from "./search-heading-find"
export {SearchHeadingNetflix}
