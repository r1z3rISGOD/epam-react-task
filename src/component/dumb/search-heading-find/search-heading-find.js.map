{"version":3,"file":"search-heading-find.js","sourceRoot":"","sources":["search-heading-find.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAC9B,sCAAmC;AAM5B,IAAM,iBAAiB,GAAsC,UAAC,EAAY;QAAV,QAAQ,cAAA;IAC7E,OAAO,CACD;QACS,QAAQ;QACb,4BAAI,SAAS,EAAC,mBAAmB,sBAAqB,CACvD,CACR,CAAA;AACH,CAAC,CAAA;AAPY,QAAA,iBAAiB,qBAO7B"}