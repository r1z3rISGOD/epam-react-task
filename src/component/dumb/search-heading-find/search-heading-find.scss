@import "./src/_variables/variables.scss";

.search__heading-2 {
  font-size: 1.5rem;
  padding-bottom: 15px;
  color: white;
  font-family: $font;
}
