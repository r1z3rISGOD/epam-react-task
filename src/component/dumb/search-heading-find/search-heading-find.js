"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.SearchHeadingFind = void 0;
var React = require("react");
require("./search-heading-find.scss");
var SearchHeadingFind = function (_a) {
    var children = _a.children;
    return (React.createElement(React.Fragment, null,
        children,
        React.createElement("h1", { className: "search__heading-2" }, "FIND YOUR MOVIE")));
};
exports.SearchHeadingFind = SearchHeadingFind;
//# sourceMappingURL=search-heading-find.js.map