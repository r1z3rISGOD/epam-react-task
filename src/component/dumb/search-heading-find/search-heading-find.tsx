import * as React from 'react'
import './search-heading-find.scss'

interface SearchHeadingFindProps {
    children: React.ReactNode
}

export const SearchHeadingFind : React.FC<SearchHeadingFindProps> = ({ children }) => {
  return (
        <>
                {children}
            <h1 className="search__heading-2">FIND YOUR MOVIE</h1>
        </>
  )
}
