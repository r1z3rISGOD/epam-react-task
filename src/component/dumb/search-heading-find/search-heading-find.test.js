import { SearchHeadingFind } from './search-heading-find.tsx'
import React from 'react'

it('search heading find', () => {
  const comp = shallow(<SearchHeadingFind/>)
  expect(comp).toMatchSnapshot()
})
