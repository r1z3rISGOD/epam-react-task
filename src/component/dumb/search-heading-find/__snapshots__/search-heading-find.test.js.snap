// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search heading find 1`] = `
<Fragment>
  <h1
    className="search__heading-2"
  >
    FIND YOUR MOVIE
  </h1>
</Fragment>
`;
